inc/namespace.php

```diff
@@ -22,6 +22,11 @@
 function bootstrap() {
 	add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_elasticpress' );
 	add_filter( 'altis_healthchecks', __NAMESPACE__ . '\\add_elasticsearch_healthcheck' );
+
+	// Load debug bar for ElasticPress if enabled in the config.
+	if ( get_config()['modules']['search']['enabled'] ?? false ) {
```

Comment:
I've been a silly - this should've been to check that `get_config()['modules']['dev-tools']['query-monitor']` is true - sorry about that. If this code is running we already know the search module is enabled.

Reply:
✅

Reply:
You removed the check altogether? I meant to change it to check that query monitor was enabled.

Reply:
OMG, now I've been a silly and totally misunderstood it. 🤦‍♀️sorry!